ledger/participant-integration-api/src/main/scala/platform/apiserver/execution/StoreBackedCommandExecutor.scala

```diff
@@ -142,6 +144,10 @@ private[apiserver] final class StoreBackedCommandExecutor(
               )
             }
 
+        case ResultNeedLocalKeyVisible(stakeholders, resume) =>
+          val visible = VisibleByKey.fromSubmitters(actAs, readAs)(stakeholders)
```

Comment:
nit/idea: If you want to make use of the fact that there is an intermediate function, you could extract `lazy val isVisible = VisibleByKey.fromSubmitters(actAs, readAs)` outside `resolveStep` and then call `isvisible(stakeholders)` to avoid the repeated execution and construction of `actAs union readAs`. But this would be entering micro-optimization territory, so take this comment with a mountain of salt.

Reply:
2c9e2aa5036c7c7e1e1607be33f9ce9a33af26dc
easy enough, thanks!